ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -220,7 +219,7 @@ export default class TestDataModal extends React.Component {
       })
       .filter(
         (item) =>
-          item.name === relatedSeries.name &&
+          item.name === relatedSeries.name.trim() &&
```

Comment:
I am able to retrieve related series without needing to `trim()`.
Are you sure this fix is needed? Could you share the Graph view links you've been using to test this?

Reply:
http://localhost:5000/perfherder/graphs?timerange=5184000&series=mozilla-central,2833204,1,6

Reply:
Thanks for the link. Looks like we need this `trim()` operation because of a backend bug.

Reply:
@esanuandra could you actually trim both names?

Reply:
I suspected as well the issue might be linked to Bug 1523037.
Sure, I will trim them both.